src/ClientSymbols/QSettingsBasedStorageManagerTest.cpp

```diff
@@ -21,15 +22,24 @@ constexpr const char* kOrgName = "The Orbit Authors";
 
 namespace orbit_client_symbols {
 
-TEST(QSettingsBasedStorageManager, LoadAndSavePaths) {
-  QCoreApplication::setOrganizationName(kOrgName);
-  QCoreApplication::setApplicationName("QSettingsBasedStorageManager.LoadAndSavePaths");
+class QSettingsBasedStorageManagerTest : public ::testing::Test {
+ protected:
+  void SetUp() override {
+    QCoreApplication::setOrganizationName(kOrgName);
 
-  {  // clear before test;
-    QSettings settings;
-    settings.clear();
+    const testing::TestInfo* test_info = testing::UnitTest::GetInstance()->current_test_info();
+    const QString test_suite_name{test_info->test_suite_name()};
+    const QString test_name{test_info->name()};
+    QCoreApplication::setApplicationName(QString("%1.%2").arg(test_suite_name).arg(test_name));
+
+    {  // clear before test;
+      QSettings settings;
+      settings.clear();
+    }
```

Comment:
Nit: The scope is not needed since it's the last thing in the function

Reply:
done